### DISCUSSION TITLE
Approximate release of " feat(export): GameObjectExportSettings.layerMask"

### DISCUSSION THREAD

**Discussion by GDGRhinox** [atteneder/glTFast#528] [Q&A] [answered]
Using GLTFast for exporting GLB files is a dream, however exporting my scene includes some debugging meshes that should be excluded. Furthermore disabling these specific GameObjects in the hierarchy is not an option for us (*GameObjectExportSettings.onlyActiveInHierarchy*).

Looking at the main branch, the following commit was made which would alleviate my problem, since I can place these objects on a different layer:
**feat(export): GameObjectExportSettings.layerMask to include or exclude GameObjects from export based on their layer**
 #[10ce87df1673586f62b2535d6820fb2bc58d8ead]

Could anyone provide an estimate when this feature will be released?

**Comment by atteneder** [comment, marked as answer]
Best case: this week
Expected: next week
Worst case: in two weeks

hth
